src/test/java/edu/harvard/iq/dataverse/api/MoveIT.java

```diff
@@ -145,6 +156,85 @@ public void testMoveDataset() {
 
     }
 
+    @Test
+    public void testMoveDatasetNotification() {
+        sendNotificationOnDatasetMoveSetting(true);
+        // Create the first user/dataverse (superuser)
+        Response createUser1 = UtilIT.createRandomUser();
+        createUser1.prettyPrint();
+        createUser1.then().assertThat()
+                .statusCode(OK.getStatusCode());
+        String user1Username = UtilIT.getUsernameFromResponse(createUser1);
+        String user1ApiToken = UtilIT.getApiTokenFromResponse(createUser1);
+        UtilIT.setSuperuserStatus(user1Username, true);
+
+        Response createDataverse1 = UtilIT.createRandomDataverse(user1ApiToken);
+        createDataverse1.prettyPrint();
+        createDataverse1.then().assertThat()
+                .statusCode(CREATED.getStatusCode());
+        String dataverseAlias1 = UtilIT.getAliasFromResponse(createDataverse1);
+
+        // Create the second user/dataverse
+        Response createUser2 = UtilIT.createRandomUser();
+        createUser2.prettyPrint();
+        createUser2.then().assertThat()
+                .statusCode(OK.getStatusCode());
+        String user2Username = UtilIT.getUsernameFromResponse(createUser2);
+        String user2ApiToken = UtilIT.getApiTokenFromResponse(createUser2);
+
+        Response createDataverse2 = UtilIT.createRandomDataverse(user2ApiToken);
+        createDataverse2.prettyPrint();
+        createDataverse2.then().assertThat()
+                .statusCode(CREATED.getStatusCode());
+        String dataverseAlias2 = UtilIT.getAliasFromResponse(createDataverse2);
+
+        // User2 creates dataset in DV2
+        Response createDataset = UtilIT.createRandomDatasetViaNativeApi(dataverseAlias2, user2ApiToken);
+        createDataset.prettyPrint();
+        createDataset.then().assertThat()
+                .statusCode(CREATED.getStatusCode());
+        Integer datasetId = UtilIT.getDatasetIdFromResponse(createDataset);
+
+        // clear existing notifications (so the DATASETMOVED notification will be the only one)
+        clearNotifications(user1ApiToken);
+        clearNotifications(user2ApiToken);
+
+        // User1(superuser) moves the dataset from dataverse2 to dataverse1
+        Response moveDataset = UtilIT.moveDataset(datasetId.toString(), dataverseAlias1, user1ApiToken);
+        moveDataset.prettyPrint();
+        moveDataset.then().assertThat()
+                .statusCode(OK.getStatusCode())
+                .body("data.message", equalTo(BundleUtil.getStringFromBundle("datasets.api.moveDataset.success")));
+
+        // verify that a notification was sent to user1
+        Response getNotifications = UtilIT.getNotifications(user1ApiToken);
+        getNotifications.prettyPrint();
+        getNotifications.then().assertThat()
+                .statusCode(OK.getStatusCode())
+                .body("data.notifications[0].type", equalTo("DATASETMOVED"))
```

Comment:
Given that this test can (currently) be run in parallel with other tests, other notifications can be created and notifications[0] may be something else.